atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -260,6 +261,34 @@ trait AtumImplicitsBase {
       dataset
     }
 
+    /**
+     * The method returns ControlMeasure object from the Atum context
+     * @return - ControlMeasure object
+     */
+    def getControlMeasure: ControlMeasure = {
+      atum.preventNotInitialized()
+      atum.controlFrameworkState.accumulator.getControlMeasure
+    }
+
+    /**
+     * The method returns AdditionalInfo object from the Atum context
+     * @return - AdditionalInfo object
+     */
+    def getAllAdditionalInfo: Map[String, String] = {
+      atum.preventNotInitialized()
+      atum.controlFrameworkState.accumulator.getControlMeasure.metadata.additionalInfo
+    }
+
+    /**
+     * The method returns Option of AdditionalInfo value for given key from the Atum context
+     * @param key - the AdditionalInfo key
+     * @return - Option[String] with value for given AdditionalInfo key
+     */
+    def getAdditionalInfo(key: String): Option[String] = {
+      atum.preventNotInitialized()
```

Comment:
Same as above, only this time the obvious choice is `getAllAdditionalInfo`.
Less code repetition and transitive effect if something in the called code changes (still good value, despite the IDE might make such refactoring much easier).